src/Executor/Usage.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace OpenRuntimes\Executor;
+
+use Exception;
+use Utopia\CLI\Console;
+use Utopia\Orchestration\Orchestration;
+use Utopia\System\System;
+
+use function Swoole\Coroutine\batch;
+
+class Usage
+{
+    public const PREFIX_HOST = 'host';
+    public const PREFIX_RUNTIME = 'runtime_';
+
+    protected ?float $hostUsage = null;
+
+    /**
+     * @var array<string,float>
+     */
+    protected array $containerUsage = [];
+
+    public function __construct(protected Orchestration $orchestration)
+    {
+    }
+
+    public function run(): void
+    {
+        $this->hostUsage = null;
```

Comment:
Instead of keeping this `null` keep `zero` as the default. so it makes calculations easier in the server